examples/resnet/user_script.py

```diff
@@ -130,7 +126,7 @@ def resnet_calibration_reader(data_dir, batch_size, *args, **kwargs):
 # keep this to demo/test custom evaluation function
 def eval_accuracy(model: OliveModelHandler, data_dir, batch_size, device, execution_providers):
```

Comment:
per https://github.com/microsoft/Olive/pull/1228/files#r1681720056, I think we could make `data_dir` and `batch_size` kwargs here.